src/Core/src/Core/IAccelerator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Maui
+{
+	public interface IAccelerator
+	{
+		/// <summary>
+		/// Specifies the virtual key used to modify another keypress. 
+		/// </summary>
+		public IEnumerable<string> Modifiers { get; }
+
+		/// <summary>
+		/// Specifies the values for each virtual key.
+		/// </summary>
+		public IEnumerable<string> Keys { get; }
```

Comment:
Chatted with @rachelkang a little on this. Seems like we can modify this `interface` to be more accurate? It seems like we're structuring this based on the `Controls` APIs which aren't completely accurate?

Reply:
updated to use just one string